src/components/search/search.tsx

```diff
@@ -155,23 +90,29 @@ const Search: React.FunctionComponent<SearchProps> = ({
                         name="Snippet"
                         label="Code Snippets"
                         onToggle={checked => onCheckToggle(checked, 'Snippet')}
-                        checked={codeLevelFilters.includes('Snippet')}
+                        checked={
+                            !!allFilters.find(filt => filt.name === 'Snippet')
+                        }
                     />
                     <Checkbox
                         name="Full Application"
                         label="Full Applications"
                         onToggle={checked =>
                             onCheckToggle(checked, 'Full Application')
                         }
-                        checked={codeLevelFilters.includes('Full Application')}
+                        checked={
+                            !!allFilters.find(
```

Comment:
small nitpick but I would have created a `const` called `isFullApplicationChecked` and referenced it here.